clang/lib/CIR/CodeGen/CIRGenBuiltinAArch64.cpp

```diff
@@ -1616,9 +1618,6 @@ mlir::Value buildNeonCall(unsigned int builtinID, CIRGenFunction &cgf,
   if (shift > 0)
     llvm_unreachable("Argument shift NYI");
 
-  if (builtinID != clang::NEON::BI__builtin_neon_vrndns_f32)
```

Comment:
get rid of this NYI as It's hard to imagine anyone would accidentally call buildNeonCall